chirp/src/chirp_fs_hdfs.c

```diff
@@ -563,7 +564,7 @@ static INT64_T chirp_fs_hdfs_rmdir(const char *path)
 {
 	RESOLVE(path)
 	debug(D_HDFS, "rmdir %s", path);
-	return hdfs_services->unlink(fs, path, 1);
+	return do_unlink(path,1);
```

Comment:
I'm inclined to agree, but I thought the existing behavior might have been deliberate.
Let's me see if it works the other way...